packages/console/src/console-history/ConsoleHistoryItemUtils.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Converts a time difference in seconds to a human-readable string.
+ * @param diff The time difference in seconds.
+ * @returns A string representing the time difference.
+ */
+export const convertedDiff = (diff: number): string => {
```

Comment:
Write as a normal function

```suggestion
export function convertedDiff(diff: number): string {
```

Reply:
done